### DISCUSSION TITLE
pymodbus.console do not start with version 3.7.0

### DISCUSSION THREAD

**Discussion by serioca** [pymodbus-dev/pymodbus#2285] [Q&A] [answered]
When I try to start pymodbus.console I get the following error message:

_ImportError: cannot import name 'ModbusBinaryFramer' from 'pymodbus.transaction' (/Users/sergiobruni/.pyenv/versions/3.12-dev/envs/panarea2/lib/python3.12/site-packages/pymodbus/transaction.py). Did you mean: 'ModbusAsciiFramer'?_

thanks

**Comment by janiversen** [comment, marked as answer]
@dhoomakethu one for you.


pymodbus.console is maintained in its own repo, which seems not to have been updated.

**Comment by janiversen** [comment]
Please create an issue in https://github.com/pymodbus-dev/repl
